overlays/bootstrap.nix

```diff
@@ -432,7 +458,8 @@ in {
                   "unix" "xhtml"
                 ];
             }];
-          } // args)).nix-tools.components.exes;
+          } // args);
+        exes = project.nix-tools.components.exes // project.hpack.components.exes;
```

Comment:
Hm, I wonder if we should instead consider `hpack` to be part of `cabal-install`? It's a little surprising to put it in here - I certainly wouldn't expect to find it there.

I guess the reason not to have it as it's own tool is just that it would be a pain to do the plumbing?

Reply:
I put it there because nix-tools actually depends on hpack, so it was already part of hsPkgs.  This also means the version used is governed by the constraints in `nix-tools.cabal` (rather than something in nix).